hpke.go

```diff
@@ -252,6 +260,10 @@ func (ctx *EncryptContext) Seal(aad, pt []byte) []byte {
 	return ct
 }
 
+func (ctx *EncryptContext) Export(context []byte, L int) []byte {
```

Comment:
You can put this method on `cipherContext` and it will automatically propagate to EncryptContext and DecryptContext.